cstore_fdw--1.5.sql

```diff
@@ -58,3 +60,37 @@ CREATE EVENT TRIGGER cstore_drop_event
     ON SQL_DROP
     EXECUTE PROCEDURE cstore_drop_trigger();
 
+CREATE TABLE public.pg_cstore_tables (logicalrelid regclass PRIMARY KEY);
+ALTER TABLE public.pg_cstore_tables SET SCHEMA pg_catalog;
+
+CREATE OR REPLACE FUNCTION public.register_cstore_table(table_name oid)
+	RETURNS boolean
+	LANGUAGE plpgsql STRICT
+	AS $rct$
+BEGIN
+    IF EXISTS(
+		SELECT fdwname FROM pg_foreign_table ft, pg_foreign_server fs, pg_foreign_data_wrapper fdw
+			WHERE 
+				ft.ftrelid = table_name AND
+				fs.oid = ft.ftserver AND
+				fdw.oid = fs.srvfdw AND
+				fdw.fdwname = 'cstore_fdw'
+				) AND
+		NOT EXISTS(SELECT * FROM pg_cstore_tables WHERE logicalrelid = table_name)
```

Comment:
For the time being, I am not dropping support for 9.4 and earlier. I am going to open an issue on moving aforementioned UDFs to c code.